ibis/file/parquet.py

```diff
@@ -0,0 +1,113 @@
+import ibis
+import ibis.expr.datatypes as dt
+import ibis.expr.operations as ops
+from ibis.file.client import FileClient
+from ibis.pandas.core import pre_execute, execute
+import pyarrow as pa
+import pyarrow.parquet as pq
+
+
+def connect(dictionary):
+        return ParquetClient(dictionary)
+
+
+_ARROW_DTYPE_TO_IBIS_TYPE = {
+    'int8': dt.int8,
+    'int16': dt.int16,
+    'int32': dt.int32,
+    'int64': dt.int64,
+    'uint8': dt.uint8,
+    'uint16': dt.uint16,
+    'uint32': dt.uint32,
+    'uint64': dt.uint64,
+    'halffloat': dt.float16,
+    'float': dt.float32,
+    'double': dt.float64,
+    'string': dt.string,
+    'binary': dt.binary,
+    'bool': dt.boolean,
+    'timestamp[ns]': dt.timestamp,
+    'timestamp[us]': dt.timestamp,
+}
+
+
+def arrow_types_to_ibis_schema(schema):
```

Comment:
Can you make a note somewhere that this doesn't handle complex types yet?

Reply:
This seems not implemented :>
```
In [5]: hash(pa.int8())
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-5-bca0e6e2f6af> in <module>()
----> 1 hash(pa.int8())

TypeError: unhashable type: 'pyarrow.lib.DataType'
```